src/core_atmosphere/dynamics/mpas_atm_time_integration.F

```diff
@@ -31,6 +31,7 @@ module atm_time_integration
    use mpas_atmphys_driver_microphysics
    use mpas_atmphys_todynamics
    use mpas_atmphys_utilities
+   use stochastic_physics_mpas, only : stochastic_physics_pattern_apply
```

Comment:
We will probably want to call this something like "mpas_atmphys_stoch_physics" to match the names of the other functions here.

Reply:
Maybe? At this point are we thinking of stochastic physics as an external library or as a part of mpas?

Reply:
I interpreted "stochastic_physics_mpas" as the function within MPAS that calls stochastic physics.  That's why I was thinking it would make sense for the function name to be consistent.  Obviously stochastic physics itself is part of the UFS repository, not part of MPAS.

Reply:
@NingWang325, Ning will modify and push commit to PR branch.